### ISSUE TITLE
'Select all' checkbox only selects all on current page

### ISSUE THREAD

**Issue by jacksos101** [samuelgrant/fight-for-kidz#221]
On any datatable with the select all checkbox, only the current page is selected.

Potential solutions:

- Do nothing
- Make the checkbox select everything even if its not visible on screen
- Add a 'copy all to group' button at the top of the page

**Comment by MitchellQ** [comment]
yes this is a known issue its the reason why we made the datatable default number 100 - in the hopes there is likely less than 100 however this is a poor fix and we could probably should do something better
